test/variables/test_variable.jl

```diff
@@ -61,6 +62,7 @@ using Test, ReactiveMP, Rocket, BayesBase, Distributions, ExponentialFamily
             activate!.(variablesmx, TestOptions())
 
             setmarginals!(variablesmx, dist)
+            @test_throws AssertionError setmarginals!(variablesmx, [dist])
```

Comment:
I missed this one. What do you mean?

Reply:
@Nimrais 
I just noticed that the purpose of this test might be a bit unclear (in the future), as it currently seems to throw errors for two reasons rather than solely due to a mismatched length. Specifically, since `dist` is a matrix and `[dist]` is an array of matrices, this combination isn't supported and may also throw. To effectively test the `length`, you can exclude the last element from `dist`. A straightforward way to achieve this is by using `dist[begin:end-1]`, which will include all elements of `dist` except the last one and ensure that the length is equal to `N - 1`.

Reply:
We do not have any multivariate distributions in this test, only this ones: NormalMeanVariance(-2.0, 3.0), NormalMeanPrecision(-2.0, 3.0), PointMass(2.0).

Reply:
Ah sorry, you are right, I confused myself with the `variablesmx` variable. The updated tests are also good!

Reply:
But I will add it now, your concern is reasonable.

Reply:
As I can see you've added it [here](https://github.com/biaslab/ReactiveMP.jl/pull/366/commits/de20edac1bceeb9b72d546e7c796ec56f6013e1f) already. This is enough :)